src/Microsoft.Fx.Portability/ObjectModel/AnalyzeRequest.cs

```diff
@@ -37,7 +37,8 @@ public sealed class AnalyzeRequest : IComparable
         public int CompareTo(object obj)
         {
             var analyzeObject = obj as AnalyzeRequest;
-            return ApplicationName.CompareTo(analyzeObject.ApplicationName);
+
+            return string.Compare(ApplicationName, analyzeObject.ApplicationName, StringComparison.CurrentCulture);
```

Comment:
I was hoping someone could comment on whether to use `StringComparison.CurrentCulture` or `StringComparison.Ordinal`.